ui_src/src/domain/administration/versionUpgrade/index.js

```diff
@@ -60,13 +68,25 @@ function VersionUpgrade() {
         }
     };
 
+    const getSystemVersion = useCallback(async () => {
```

Comment:
We should save the current version when the application is up; right now, we check if the version is the latest and keep it in a global state, so the best will be to save the current version in a global state and use the state here.
The implementation of it appears in the sidebar component.

Reply:
Based on my understanding, it seems necessary to save the current version of the application in the global state. Then, whenever there is a need for version-related tasks, we can make a request to the Github releases to retrieve the latest version and check if it's the most recent. If it's not, we will display an update button instead of the current approach, where we only save the "isLatest" flag in the global state.

Since the current implementation is utilized in multiple places, I adopted the same method.

Reply:
GET_CLUSTER_INFO is called from the sideBar and signup component. The signup component appears once in the system's entire life and is not the right place to add this logic.

The is_latest flag is saved in the global state from the sideBar component, so you can also add saving the current version in the global state.